tensorflow/compiler/mlir/xla/transforms/rewriters.h

```diff
@@ -38,8 +41,9 @@ void PopulateXlaToStdPatterns(OwningRewritePatternList *patterns,
                               MLIRContext *ctx);
 
 // Collection of rewrite patterns for lowering of HLO to LHLO dialect.
-void populateHLOToLHLOConversionPattern(MLIRContext *context,
-                                        OwningRewritePatternList *patterns);
+void populateHLOToLHLOConversionPattern(
```

Comment:
`third_party/tf_runtime/tools/tf_kernel_gen/kernel_lowering.cc:98:72: error: too few arguments to function call, expected 3, have 2
    ::mlir::xla_hlo::populateHLOToLHLOConversionPattern(&ctx, &patterns);
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~                ^
./third_party/tensorflow/compiler/mlir/xla/transforms/rewriters.h:44:6: note: 'populateHLOToLHLOConversionPattern' declared here
void populateHLOToLHLOConversionPattern(`

Reply:
@dfki-ehna can you please check above error ?

Reply:
@rthadur we have fixed this error locally. However, [Buffer Assignment](https://github.com/tensorflow/tensorflow/pull/37212) has been reverted. We are opening a new PR for Buffer Assignment and adding new features and tests. We should wait for getting it landed again.

Reply:
thank you, do you have a PR number ?

Reply:
I don't think it was reverted.